src/main/java/com/google/sps/servlets/RepSubmitQuestionnaireServlet.java

```diff
@@ -50,10 +50,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
         List<String> topicList = Arrays.asList(topics.split(","));
         topicList.replaceAll(s -> repName.replaceAll("\\s+","") + s);
         String platforms = request.getParameter(PLATFORM_LIST);
-        List<String> platformList = Arrays.asList(platforms.split("\\*,"));
-        //Remove the * from the string of the last platform
-        platformList.get(platformList.size() - 1).replace("\\*", "");
-       
+        List<String> platformList = new ArrayList<String>(Arrays.asList(platforms.split("\\*,")));
```

Comment:
same comment, do you need this new ArrayList<String>. Type inference should work? Also add the comment back.